package.json

```diff
@@ -57,6 +57,7 @@
     "rollup-plugin-uglify": "^1.0.1",
     "rollup-plugin-vue": "^2.3.1",
     "rollup-watch": "^3.2.2",
+    "vue": "2.2.5",
```

Comment:
nuxt depends on an older vue-template compiler with lower semver/minor version & Vue requires exact matching version to work, however this is just for development and docs generation. Thia was the only workaround i had found.